WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -47,6 +52,49 @@ struct CollapsibleProductRowCard: View {
                 Text(Localization.priceLabel)
                 CollapsibleProductCardPriceSummary(viewModel: viewModel)
             }
+            .padding(.bottom)
+            HStack {
+                if viewModel.discount == nil {
+                    Button(Localization.addDiscountLabel) {
+                        onAddDiscount()
+                    }
+                    .buttonStyle(PlusButtonStyle())
+                } else {
+                    HStack {
+                        Button(action: {
+                            onAddDiscount()
+                        }, label: {
+                            HStack {
+                                Text(Localization.discountLabel)
+                                Image(uiImage: .pencilImage)
+                                    .resizable()
+                                    .frame(width: Layout.iconSize, height: Layout.iconSize)
+                            }
+                        })
+                        Spacer()
+                        if let discountLabel = viewModel.discountLabel {
+                            Text(minusSign + discountLabel)
```

Comment:
Will this localize properly for rtl languages? I'm not sure but I don't think it will. Perhaps using a number formatter to make it negative would be better?

Reply:
That's an interesting one, I thought the symbol should be on the opposite side, but that doesn't seem to be the case:

```
Even though the language is read from right to left, the numerical and financial contexts such as prices, especially involving arithmetic symbols, maintain a left-to-right order. This is recognized and understood in RTL languages and cultures.
```

We seem to not deal with this correctly across the app, but I would need to test it further to confirm. This is a quick test of how refunds show with an RTL currency:
<img width=500 src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/5142c9c6-7924-4db3-9d7d-36c3721789ff">

Since the value from `discountLabel` is already currency-formatted (but as an absolute value) I tested a couple of approaches like using the existing `isNegative` property, but this still renders the symbol in the wrong place, since changes the position of the `-` symbol either in the view or in the CurrencyFormatter. I've opened a different issue here to investigate further: https://github.com/woocommerce/woocommerce-ios/issues/10913